src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,42 @@
+footer {
+  width: 100%;
+  height: fit-content;
+  margin-top: 3rem;
+  background-color: #353535;
```

Comment:
you can use CSS variables for this too.

Reply:
Oh.. I forget about it. Thank for noticing, I will change